tests/scripts/all.sh

```diff
@@ -344,7 +344,17 @@ fi
 if uname -a | grep -F x86_64 >/dev/null; then
 msg "build: i386, make, gcc" # ~ 30s
 cleanup
-CC=gcc CFLAGS='-Werror -m32' make
+CC=gcc CFLAGS='-Werror -Wall -Wextra -m32' make
+
+msg "test: i386, make, gcc"
+make test
```

Comment:
In 1.3, `make test` only works with CMake, not with the original Makefile.

I'd prefer to resolve this by adding `test` as a target to the Makefile, like in the 2.1 and development branches.

```
diff --git a/Makefile b/Makefile
index 441d1f9..dd4e5d6 100644
--- a/Makefile
+++ b/Makefile
@@ -60,6 +60,8 @@ clean:
 check: tests
        $(MAKE) -C tests check
 
+test: check
+
 test-ref-configs:
        tests/scripts/test-ref-configs.pl
```

Reply:
Ok, sorry about that. I will add it